diem-unified-charter.md

```diff
@@ -1,15 +1,11 @@
 # Introduction
 
-An emblem is a device, symbol, or figure adopted and used as an identifying mark.
-In culture, emblems such as a flag, badge or coat of arms communicate group identity.
-Such emblems are often comprised of a set of attributes each with symbolic meaning.
-In speech, emblems are specific nonverbal gestures or signals that have a direct verbal translation and are widely understood within a particular culture or community.
-Under International Humanitarian Law (IHL), the Red Cross, Red Crescent, and Red Crystal emblems are symbols of protection.
-Similarly, the "blue shield" is another IHL emblem that marks cultural property under special protection.
-ISO 7010 defines a set of emblems that can be used to identify hazards, these include the skull and crossbones for toxic material, the ionizing radiation symbol and the biological hazard symbol.
-Today often these emblems/symbols require a sense of sight, or touch to become known to the receiver.
-
-There is a need to sense emblems/symbols through digital communication channels.
+Emblems such as the Red Cross, Red Crescent, and Red Crystal can be symbols of
+protection governed by International Humanitarian Law (IHL).  Emblems can also
+be used to identify hazards via the skull and crossbones symbol for toxic
+materials or the ionizing radiation symbol for radiation defined by ISO 7010.
+
+There is a need to sense emblems through digital communication channels.
```

Comment:
```suggestion
There is a need to present emblems through digital communication channels.
```

Overall, I think it's good to stay more succinct here. But I think this sentence is now missing some context ("sense" was introduced previously but is not anymore). I think "present" is a more intuitive word here that shouldn't require definition.